cypress.config.ts

```diff
@@ -0,0 +1,20 @@
+import { defineConfig } from 'cypress'
+
+export default defineConfig({
+  video: false,
+  fixturesFolder: false,
+  projectId: 'r9de4a',
```

Comment:
I don't think it hurts to setup cypress. Here is the project id for a new project in Cypress: `ez44c6`
And I added the corresponding `CYPRESS_RECORD_KEY` to the repo's secrets.

Reply:
Once you've recorded the first test run in Cypress I can do a little more setup so it is more integrated with GitHub.

I'd also note that @emcelroy was briefly exploring the idea of using Cypress to test the multidata plugin embedded inside of CODAP.  I think the hardest part of doing that is getting the local webserver to run via https in GitHub actions, that will be needed so it can be embedded inside of the deployed version of CODAP. Otherwise the GitHub action would also have to download and build all of CODAP so it could run locally using http. We know how to run https dev webservers locally on our macs, but I don't think we've ever tried doing it in GitHub actions.

Reply:
@nstclair-cc is working to use cyrpess to test the WHO plugin within Codap, so this is also relevant info for her.